bin/test

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+# Unit tests
+docker-compose \
+    --project-name elm-lmp-api \
+    --file docker/docker-compose.test.yml \
+    run unit-test
```

Comment:
I think the `bin/` folder name came from the original Defra boilerplate. I agree `scripts/` is a better name - will change this separately.

Perhaps caching warrants a conversation. I've structured the `Dockerfile` to maximise use of the Docker cache to speed up builds in development.

I vaguely recall issues with Docker cache a couple of years ago but I think that might have when the COPY command didn't check for changes in folder contents so always used the cache. I haven't had issues with the cache for well over a year now, so I'd be inclined not to prevent cache use unnecessarily. Individual developers can always clear out their cache if problems do arise.